docs/semgrep-code/overview.md

```diff
@@ -0,0 +1,43 @@
+---
+slug: overview
+append_help_link: true
+title: Overview
+hide_title: false
+description: Learn about Code, a static application security testing (SAST) tool |
+    designed to find security vulnerabilities in your applications.
```

Comment:
```suggestion
    designed to find security vulnerabilities in your first-party code.
```
 
We can differentiate products through their scan targets:

Semgrep Code - first-party code
Semgrep Supply Chain - third-party code

"applications" could imply both targets but Code specifically scans only 1st party, not the whole "application"